src/common/framework/test_group.ts

```diff
@@ -223,8 +276,41 @@ class RunCaseSpecific<F extends Fixture> implements RunCase {
       // An error from init or test may have been a SkipTestCase.
       // An error from finalize may have been an eventualAsyncExpectation failure
       // or unexpected validation/OOM error from the GPUDevice.
+      if (throwSkip && ex instanceof SkipTestCase) {
+        throw ex;
+      }
       rec.threw(ex);
     }
+  }
+
+  async run(rec: TestCaseRecorder): Promise<void> {
+    rec.start();
+    if (this.subParamGen) {
+      let totalCount = 0;
+      let skipCount = 0;
+      for (const subParams of this.subParamGen(this.params)) {
+        rec.debug(new Error('subcase: ' + stringifyPublicParamsUniquely(subParams)));
```

Comment:
Perhaps we should have a different log level for this so these logs don't get hidden by default.